client/me/help/help-contact/index.jsx

```diff
@@ -437,6 +439,11 @@ const HelpContact = React.createClass( {
 		return (
 			<Main className="help-contact">
 				<HeaderCake onClick={ this.backToHelp } isCompact={ true }>{ this.translate( 'Contact Us' ) }</HeaderCake>
+				{ ! user().get().email_verified &&
```

Comment:
We can grab the user from the redux store in the `connect` below, then pass this in as a prop:

```js
export default connect(
 	( state ) => {
 		const currentUser = getCurrentUser( state );
 		return {
 			olarkTimedOut: isOlarkTimedOut( state ),
 			isEmailVerified: currentUser.email_verified,
 		};
 	}
 )( HelpContact );
```